src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/coding/missingctor/Example1.java

```diff
@@ -22,7 +23,7 @@ class ExampleDefaultCtor { // OK
 class InvalidExample { // violation, class must have a constructor.
   public void test() {}
 }
-abstract class AbstractExample { // OK
```

Comment:
Please update violation line above to have violation message `// violation 'Class should define a constructor'`

Reply:
Done.